gulpfile.js

```diff
@@ -89,17 +87,15 @@ mach.transpileLESS = function (src, debug) {
     // paths: [ path.join(__dirname, 'less', 'includes') ]
   }
   return gulp.src(src)
-    .pipe(less({
-      plugins: [autoprefix]
-    }))
     .pipe(lesshint({
       configPath: './.lesshintrc' // Options
     }))
+    .pipe(less({
+      plugins: [autoprefix]
+    }))
     .pipe(lesshint.reporter()) // Leave empty to use the default, "stylish"
     .pipe(lesshint.failOnError()) // Use this to fail the task on lint errors
     .pipe(srcmaps.init())
-    .pipe(less(opts))
```

Comment:
Please remove line 87-90. We don't need it.

Reply:
Those are kept for reference only

Reply:
It's commented code. We're better off without it :)